src/achievements/AchievementRegistry.ts

```diff
@@ -15,16 +15,17 @@ export function registrableAchievement(target: AchievementType) {
 export * as allAchievements from "./Achievements"
 
 // ---- AchievementRegistry ----
+const LOG = logger();
+
 export class AchievementRegistry {
     private readonly achievements: { [key: string]: Achievement<any> } = {};
 
     constructor(achievements: Achievement<any>[]) {
-        Util.log("info", `Registering achievements: [${achievements.map(value => value.name).join(", ")}].`);
+        LOG.info(`Registering achievements: [${achievements.map(value => value.name).join(", ")}].`)
```

Comment:
semicolon went missing (common mistake since JS does semicolon insertion and therefore rarely produces an error, we should probably put this into the linter config for consistency anyway https://palantir.github.io/tslint/rules/semicolon/)